test/gtest/graphapi_capi_mha_backward.cpp

```diff
@@ -0,0 +1,430 @@
+        MakePointwise(MIOPEN_POINTWISE_EXP, sub0, DescriptorWrapperPtr(), exp0);
+        MakePointwise(
+            MIOPEN_POINTWISE_MUL, exp0, m_realTensorMap[miopenTensorMhaZInv]->m_gapiDesc, mult0);
+
+        auto rnd = MakeGapiVirtualTensorDesc(m_testN, m_testH, m_testS, m_testS);
+        MakeRNG(m_bernulliProbability,
+                m_realTensorMap[miopenTensorMhaDropoutSeed]->m_gapiDesc,
+                m_realTensorMap[miopenTensorMhaDropoutOffset]->m_gapiDesc,
+                rnd);
+
+        DescriptorWrapperPtr mult1;
+        DescriptorWrapperPtr pwS3, pwS4;
+        MakePointwise(MIOPEN_POINTWISE_MUL, mult0, rnd, mult1);
+
+        // _TODO there is a 1/(1-p) probability on the picture
+        MakePointwise(MIOPEN_POINTWISE_MUL,
+                      mult1,
+                      m_realTensorMap[miopenTensorMhaDropoutProbability]->m_gapiDesc,
+                      pwS3);
+        MakePointwise(
+            MIOPEN_POINTWISE_MUL, pwS3, m_realTensorMap[miopenTensorMhaScaleS]->m_gapiDesc, pwS4);
+
+        //////reshape transpose on a scheme is here! ////////
+
+        auto mm1 = MakeGapiVirtualTensorDesc(m_testN, m_testH, m_testS, m_testD);
+        MakeMatmul(pwS4, m_realTensorMap[miopenTensorMhaDO]->m_gapiDesc, mm1);
+        DescriptorWrapperPtr pwS5, pwS6;
+        MakePointwise(
+            MIOPEN_POINTWISE_MUL, mm1, m_realTensorMap[miopenTensorMhaDescaleS]->m_gapiDesc, pwS5);
+        MakePointwise(MIOPEN_POINTWISE_MUL,
+                      pwS5,
+                      m_realTensorMap[miopenTensorMhaDescaleDO]->m_gapiDesc,
+                      pwS6);
+
+        MakeReduction(
+            MIOPEN_REDUCE_TENSOR_MAX, pwS6, m_realTensorMap[miopenTensorMhaAmaxDV]->m_gapiDesc);
+        MakePointwise(MIOPEN_POINTWISE_MUL,
+                      pwS6,
+                      m_realTensorMap[miopenTensorMhaScaleDV]->m_gapiDesc,
+                      m_realTensorMap[miopenTensorMhaDV]->m_gapiDesc);
+
+        ////////////////// Center-top, Right-top //////////////////////////////////
+        ///////////////////////////////////////////////////////////////////////////
+        auto mm2 = MakeGapiVirtualTensorDesc(m_testN, m_testH, m_testS, m_testS);
+        MakeMatmul(m_realTensorMap[miopenTensorMhaDO]->m_gapiDesc,
+                   m_realTensorMap[miopenTensorMhaV]->m_gapiDesc,
+                   mm2);
+
+        DescriptorWrapperPtr pwS7, pwS8, pwS9, pwS10;
+        MakePointwise(
+            MIOPEN_POINTWISE_MUL, mm2, m_realTensorMap[miopenTensorMhaDescaleDO]->m_gapiDesc, pwS7);
+        MakePointwise(
+            MIOPEN_POINTWISE_MUL, pwS7, m_realTensorMap[miopenTensorMhaDescaleV]->m_gapiDesc, pwS8);
+        MakePointwise(MIOPEN_POINTWISE_MUL, pwS8, rnd, pwS9);
+
+        MakePointwise(MIOPEN_POINTWISE_MUL,
+                      pwS9,
+                      m_realTensorMap[miopenTensorMhaDropoutProbability]->m_gapiDesc,
+                      pwS10);
```

Comment:
This part is also missing in CuDNN's version as in the comment above [CuDNN code](https://github.com/NVIDIA/cudnn-frontend/blob/98ca4e1941fe3263f128f74f10063a3ea35c7019/include/cudnn_frontend/node/sdpa_fp8_bwd.h#L261-L265)